src/Bedrock.Framework.Experimental/Protocols/Memcached/MemcachedMessageReader.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Bedrock.Framework.Infrastructure;
+using Bedrock.Framework.Protocols;
+using System;
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Text;
+using static Bedrock.Framework.Experimental.Protocols.Memcached.Enums;
+
+namespace Bedrock.Framework.Experimental.Protocols.Memcached
+{
+    public class MemcachedMessageReader : IMessageReader<MemcachedResponse>
+    {
+        public MemcachedResponse InProgressResponse { get; private set; } = new MemcachedResponse();
```

Comment:
Why do you need this as a field? Seems like it's not really persisting anything useful. You either parse the entire message everytime or you don't.

Reply:
Right.
Fixed